### ISSUE TITLE
main.d.ts referencing incorrect interface filenames

### ISSUE THREAD

**Issue by entee7** [ceolter/ag-grid#1647]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ceolter/ag-grid/blob/master/CONTRIBUTING.md#question
```

**Current behavior**

example:
export { IRowNodeStage, StageExecuteParams } from "./dist/lib/interfaces/**IRowNodeStage**";
actual file name is **iRowNodeStage**

same applicable for other references

**Expected behavior**

**Minimal reproduction of the problem with instructions**

**What is the motivation / use case for changing the behavior?**

webpack wont compile
**Please tell us about your environment:**

* **ag-Grid version:** 10.0.0

* **Browser:**

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by dmarginian** [comment]
Same issue:

[ERROR] Error at /home/david/dev/projects/ms-ng2-common-modules/simple-table/node_modules/ag-grid/main.d.ts:95:34: Cannot find module './dist/lib/interfaces/IRangeController'.
[ERROR] Error at /home/david/dev/projects/ms-ng2-common-modules/simple-table/node_modules/ag-grid/main.d.ts:127:33: Cannot find module './dist/lib/interfaces/IAggFuncService'.
[ERROR] Error at /home/david/dev/projects/ms-ng2-common-modules/simple-table/node_modules/ag-grid/main.d.ts:128:35: Cannot find module './dist/lib/interfaces/IClipboardService'.
[ERROR] Error at /home/david/dev/projects/ms-ng2-common-modules/simple-table/node_modules/ag-grid/main.d.ts:129:31: Cannot find module './dist/lib/interfaces/IExcelCreator'.
[ERROR] Error at /home/david/dev/projects/ms-ng2-common-modules/simple-table/node_modules/ag-grid/main.d.ts:130:30: Cannot find module './dist/lib/interfaces/IMenuFactory'.
[ERROR] Error at /home/david/dev/projects/ms-ng2-common-modules/simple-table/node_modules/ag-grid/main.d.ts:137:37: Cannot find module './dist/lib/interfaces/IViewportDatasource'.
[ERROR] Error at /home/david/dev/projects/ms-ng2-common-modules/simple-table/node_modules/ag-grid/main.d.ts:138:37: Cannot find module './dist/lib/interfaces/IContextMenuFactory'.
[ERROR] Error at /home/david/dev/projects/ms-ng2-common-modules/simple-table/node_modules/ag-grid/main.d.ts:139:30: Cannot find module './dist/lib/interfaces/ICompFactory'.
[ERROR] Error at /home/david/dev/projects/ms-ng2-common-modules/simple-table/node_modules/ag-grid/main.d.ts:140:51: Cannot find module './dist/lib/interfaces/IRowNodeStage'.

**Comment by nicke** [comment]
it seams to be an problem on case sensitive filesystems (e.g. on Linux)

- export { IRangeController } from "./dist/lib/interfaces/IRangeController";
+ export { IRangeController } from "./dist/lib/interfaces/iRangeController";
...

**Comment by andreasgebauer** [comment]
I agree with @nicke. Compiles without errors on Mac OS and Windows. My machine and Jenkins running on Linux are having issues.

**Comment by makinggoodsoftware** [comment]
Fixed in 10.0.1
